plugins-compat-tester/pom.xml

```diff
@@ -59,16 +54,20 @@
       <version>1.10.7</version>
     </dependency>
 
-    <!-- Upper bounds in Embedder -->
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-utils</artifactId>
       <version>3.2.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-container-default</artifactId>
```

Comment:
We can use it since we do not longer bundle Maven libs.
http://olamy.blogspot.com/2010/09/use-maven-3-apis-in-hudson-plugin.html from @olamy